src/sentry/static/sentry/app/components/assigneeSelector.jsx

```diff
@@ -312,12 +312,24 @@ const AssigneeSelector = createReactClass({
                     <span className="icon-circle-cross" /> {t('Clear Assignee')}
                   </MenuItem>
                 )}
+
               {!memberListLoading && (
                 <li>
                   <ul>{[...teamNodes, ...memberNodes]}</ul>
                 </li>
               )}
 
+              {ConfigStore.getConfig().invitesEnabled && (
```

Comment:
when would and wouldn't this be set?

Reply:
This is just deriving from a config setting https://github.com/getsentry/sentry/blob/master/src/sentry/templatetags/sentry_react.py#L128